### PR TITLE
Add blog post about Map Local add-on

### PR DESCRIPTION
**Pull request by Keindel**
please take a look whether the description is good enough

### CODE REVIEW

File: site/content/blog/2023-10-12-maplocal-add-on/index.md
Comment (kingthorin) [inline site/content/blog/2023-10-12-maplocal-add-on/index.md:26]: RMB?

File: site/content/blog/2023-10-12-maplocal-add-on/index.md
Comment (kingthorin) [inline site/content/blog/2023-10-12-maplocal-add-on/index.md:32]: URLs

Comment (kingthorin) [inline site/content/blog/2023-10-12-maplocal-add-on/index.md:32]: Maybe include a screen shot.

A small video or animated gif of an example in action (before/after of a site)

**Review by kingthorin — COMMENTED** [review]
Maybe add a bit about your motivation to create the add-on? Maybe outline some of your use cases?

**Comment by kingthorin** [comment]
To address the DCO requirement you'll need to sign-off the commit(s):

- https://github.com/zaproxy/zaproxy/blob/main/CONTRIBUTING.md#developer-certificate-of-origin
- https://git-scm.com/docs/git-commit#Documentation/git-commit.txt---signoff

File: site/content/blog/2023-10-12-maplocal-add-on/index.md
Comment (psiinon) [inline site/content/blog/2023-10-12-maplocal-add-on/index.md:10]: We're publishing a blog post about 2.14.0 today, so it might be worth holding off for a couple of days 😉

File: site/content/blog/2023-10-12-maplocal-add-on/index.md
Comment (psiinon) [inline site/content/blog/2023-10-12-maplocal-add-on/index.md:15]: Go into a bit more detail for why someone might want to use it?
Esp someone who has not used the same feature in Charles or Proxyman before.

File: site/content/blog/2023-10-12-maplocal-add-on/index.md
Comment (psiinon) [inline site/content/blog/2023-10-12-maplocal-add-on/index.md:25]: A screenshot of the dialog would help here

File: site/content/blog/2023-10-12-maplocal-add-on/index.md
Comment (psiinon) [inline site/content/blog/2023-10-12-maplocal-add-on/index.md:12]: Add your name to https://github.com/zaproxy/zaproxy-website/blob/main/site/data/authors.yml and reference that

**Review by psiinon — CHANGES_REQUESTED** [review]

**Comment by Keindel** [comment]
changes applied, expanations and images added

**Comment by psiinon** [comment]
Looking good, notwithstanding the minor comments already mentioned 😁

**Comment by psiinon** [comment]
The date needs to change to today - in the folder name as well..

File: site/content/blog/2023-10-31-maplocal-add-on/index.md
Comment (thc202) [inline site/content/blog/2023-10-31-maplocal-add-on/index.md:5]: Remove?

**Review by psiinon — APPROVED** [review]

**Review by kingthorin — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

**Pull request merged**
